### ISSUE TITLE
Stuck on copying file into the mtp device

### ISSUE THREAD

**Issue by c02y** [boydaihungst/simple-mtpfs.yazi#2]
I mount a mtp device, jump into in folder in the mtp device, create a new tab in yazi, navigate into another folder on my system, copy a 10MB file, try to paste the file into the mtp device, it got stuck on the process and the process never finished. And the whole yazi got stuck when I try to switch to the other tab, I cannot move my point in yazi, then I have to quit yazi by force, kill simple-mtpfs and `sudo umount /home/user/Media/xxx` to start using simple-mtpfs in yazi again.

Let me know what should I do if you need any log or debug info.

-------------------------
**UPDATE**:
it seems it is the issue of simple-mtpfs binary itself, I use simple-mtpfs without yazi, `cp` command will also get stuck and fail with `cp: error writing './thefile': Input/output error` after quite some time.

**Comment by boydaihungst** [comment]
Related to this issue: https://github.com/phatina/simple-mtpfs/issues/80

They said that u could install older version of simple-mtpfs to fix this:
https://github.com/phatina/simple-mtpfs/commit/e1215568516cc50b8165bb99dc9d37b96b54c32f

You could install the older version of `simple-mtpfs` with the following commands:
```sh
git clone https://github.com/phatina/simple-mtpfs && cd simple-mtpfs
git checkout e1215568516cc50b8165bb99dc9d37b96b54c32f
mkdir build && ./autogen.sh && make 
sudo make install
```

**Comment by c02y** [comment]
> Related to this issue: [phatina/simple-mtpfs#80](https://github.com/phatina/simple-mtpfs/issues/80)
> 
> They said that u could install older version of simple-mtpfs to fix this: [phatina/simple-mtpfs@e121556](https://github.com/phatina/simple-mtpfs/commit/e1215568516cc50b8165bb99dc9d37b96b54c32f)
> 
> You could install the older version of `simple-mtpfs` with the following commands:
> 
> ```shell
> git clone https://github.com/phatina/simple-mtpfs && cd simple-mtpfs
> git checkout e1215568516cc50b8165bb99dc9d37b96b54c32f
> mkdir build && ./autogen.sh && make 
> sudo make install
> ```

Thanks, the compile and install commands you provided failed, it should be:
```
$ mkdir build && cd build
$ ../configure
$ make
$ sudo make install
```

I tried it in yazi, copy works fine. The only issue is that, the size of the file displayed is `0B`, it is not `0B` actually, and the `Spot` saying the file is `Mimetype:   inode/empty`, if I cd into the directory in my terminal, `ls -l`, the size is expected, and the file can be opened inside my mtp device.

Another issue is, if I delete the file from the mtp device, the file is still inside yazi, I'll file another issue about this.

--------------------
**UPDATE**:
The size is normal again, nevermind.
